x-pack/plugins/security_solution/public/dashboards/pages/landing_page/index.tsx

```diff
@@ -134,19 +134,18 @@ export const DashboardsLandingPage = () => {
 
       {canReadDashboard && securityTagsExist && initialFilter ? (
         <>
+          <EuiTitle size="xxxs">
```

Comment:
NIT: can we add some more space on top? the title looks very close to the bottom previous section
```suggestion
          <EuiSpacer size="m" />
          <EuiTitle size="xxxs">
```
(not sure if "m" is too much)